src/main/java/mate/academy/model/User.java

```diff
@@ -53,8 +57,25 @@ public void setSalt(byte[] salt) {
     @Override
     public String toString() {
         return "User{"
-            + "id=" + id
-            + ", email='" + email + '\''
-            + '}';
+                + "id=" + id
+                + ", email='"
+                + email + '\''
+                + '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        User user = (User) o;
+        return Objects.equals(id, user.id)
+                && Objects.equals(email, user.email)
+                && Objects.equals(password, user.password);
```

Comment:
Including the `password` field in the `equals` method might not be ideal, especially if passwords are hashed and salted. Consider removing the `password` field from the `equals` method to avoid potential security issues.